tests/matrix_free/advect_1d.cc

```diff
@@ -112,10 +112,10 @@ class MatrixFreeAdvectionBasic
       {
         phi.reinit(cell);
         phi.read_dof_values(src);
-        phi.evaluate(true, false);
+        phi.evaluate(EvaluationFlags::values);
         for (unsigned int q = 0; q < phi.n_q_points; ++q)
           phi.submit_gradient(advection * phi.get_value(q), q);
-        phi.integrate(false, true);
+        phi.integrate(EvaluationFlags::gradients);
```

Comment:
That's charming from the point of documenting itself, isn't it? :+1:

Reply:
much better, yes. :-)